ScanDemoExample/scanlibrary/src/main/java/com/scanlibrary/PolygonView.java

```diff
@@ -238,7 +238,11 @@ public boolean onTouch(View v, MotionEvent event) {
                 default:
                     break;
             }
-            polygonView.invalidate();
+            if (eid == MotionEvent.ACTION_MOVE) {
```

Comment:
Thank you for the PR. Could you please help me understand the reason behind these changes.